k4Reco/ConformalTracking/components/ConformalTracking.cpp

```diff
@@ -83,7 +83,7 @@ ConformalTracking::ConformalTracking(const std::string& name, ISvcLocator* svcLo
                       KeyValues("RelationsNames", {}),
                   },
                   {
-                      KeyValues("SiTrackCollectionName", {"CATracks"}),
+                      KeyValue("SiTrackCollectionName", "CATracks"),
```

Comment:
Does this also require changes in the python helper bits that simplify the configuration? Or would that need to be done downstream by users themselves?

Reply:
No because this only affects input and output collections and the utils in this repo are only modifying the parameters.